backend/code_review_backend/issues/management/commands/cleanup_issues.py

```diff
@@ -28,7 +34,20 @@ def add_arguments(self, parser):
             default=30,
         )
 
+    def cleanup_repositories(self):
+        unused_repositories = Repository.objects.filter(
+            base_revisions__isnull=True,
+            head_revisions__isnull=True,
+            diffs__isnull=True,
+        )
+        return unused_repositories._raw_delete(unused_repositories.db)
+
     def handle(self, *args, **options):
+        stats = defaultdict(int)
+        repo_count = self.cleanup_repositories()
+        if repo_count:
+            stats["Repository"] = repo_count
```

Comment:
You can directly log all the repositories deleted, otherwise the log at end may not be reached if no issues is found